### PR TITLE
User guide - internship application

### LINKED ISSUE #55: Update User guide

### LINKED ISSUE #65: Update commands for internship application in the User Guide for v1.1

### PR DESCRIPTION
**Pull request by keanecjy**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,176 +3,81 @@ layout: page
...
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
+**Already created the internship before?**
 
-Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+Fret not, we have an alternative method to apply for an internship! This format adds an internship with the company
+name and job name only.
```
Comment (ZoroarkDarkrai) [inline docs/UserGuide.md]: Could be confusing saying job name because the format actually uses the position of internship in the company?

Comment (keanecjy) [reply docs/UserGuide.md]: The job list in the company has the name of the job, so I don't think this will be an issue.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -77,9 +76,86 @@ Profile | me | category, descriptors
...
+#### Adding an internship application: `add-int`
+
+Applies for an internship. This internship will be automatically added to the list of jobs in the company
+(if company has already been created). Otherwise, it creates the company and adds this job into it.
+
+Format: `add -int n/COMPANY_NAME j/JOB_NAME [i/INDUSTRY] [p/PERIOD] [w/WAGE] [r/REQUIREMENT]...`
+
+Examples:
+- add -int n/Google i/Software j/Software Engineer
+- add -int n/Google i/Software j/Software Engineer r/React w/3000 r/HTML5
```
Comment (shawn-nyk) [inline docs/UserGuide.md]: Should we markdown these lines with backticks as well, to put them in code format, as per the other examples?

**Review by shawn-nyk — COMMENTED** [review]
Looks great overall, and perhaps we can address the small part highlighted before we merge!

File: docs/UserGuide.md (hunk 2 above)
Comment (keanecjy) [reply docs/UserGuide.md]: Good catch, updated it already. Thanks

**Review by shawn-nyk — APPROVED** [review]
LGTM!

**Pull request merged**